helpers/config.py

```diff
@@ -31,10 +32,34 @@ class ProjectConfig:
 
     config = None
 
-    def __init__(self, config, environment):
-        if os.path.exists(config):
-            f = yaml.load(open(config).read())
-            self.config = f[environment]
+    def __init__(self, config, environment, passwords=None):
+        self.config = self.load_yaml(config)[environment]
+        if passwords is not None:
```

Comment:
I'd change this to either just 'if passwords' or 'if isinstance(passwords, basestring)'